lib/jbuild

```diff
@@ -0,0 +1,20 @@
+(jbuild_version 1)
+
+(library
+ ((name cohttp)
+  (public_name cohttp)
+  (preprocess (pps (ppx_fields_conv ppx_sexp_conv)))
+  (libraries (re.emacs
+              stringext
+              uri
+              fieldslib
+              sexplib
+              ppx_fields_conv
```

Comment:
this is surprising, I don't suppose cohttp uses the the ppx rewriter itself at runtime?